### ISSUE TITLE
User story - Project contributors

### ISSUE THREAD

**Issue by stephendawsondev** [stephendawsondev/StackPortfolio#24]
### If applicable, add a related Epic.

Epic #9 

### User Story

As a **user**, I can **add project contributors to a project** so that I can **show who worked on the project with me**.

### Acceptance Criteria

- When another user is added to a project, they should be able to accept or reject the additional
- If a user accepts to be a contributors, the tech of that project should show up on their profile
- If a user accepts to be a contributor, the project should also appear on their profile
- When adding a contributor, a user should be able to search through their connections
- If a contributor is not in a user's connections, they should be able to invite them to connect or join the platform

### Please ensure you've completed these tasks before marking the issue as done.

- [ ] Reviewed the user story for clarity and completeness.
- [ ] Identified and linked related issues or epics.
- [ ] Created or updated the necessary documentation.
- [ ] Written unit and/or integration tests for functionality.
- [ ] All code changes have been manually tested to ensure they meet the acceptance criteria.
- [ ] Updated the issue with any relevant notes or feedback (to date).
- [ ] Merged into the main branch and deployed to production (if applicable).

### Please add any further tasks that need to be completed before the issue can be marked as done.

- [ ] Projects would need an intermediary table as the relationship will change from "one to many" to "many to many"
- [ ] UI will need to be updated to display contributors on projects
